frontend/src/components/ExternalDb.vue

```diff
@@ -37,6 +42,21 @@ export default {
       components: state => state.externalDb.components,
       externalDb: state => state.externalDb.externalDb,
     }),
+    compGroupedByModel() {
+      const result = this.components.reduce((r, a) => { // eslint-disable-next-line
+        r[a.model] = r[a.model] || [];
+        r[a.model].push(a);
+        return r;
+      }, Object.create(null));
```

Comment:
Question: why do you use a null object here, rather than just `{}`?

Reply:
`Object.create(null)` creates an object that doesn't inherit anything. But you are right, here the object to be create is not a map, so it is more neat to just use `{}`. I will change it.

Reply:
Ok, thanks for the explanation! I just got curious :)